README.md

```diff
@@ -1,10 +1,10 @@
 # What is this?
 
-This project describes a set of job titles for software engineers, operations admins and engineers and their managers. It was developed by members of the engineering team at [Urban Airship](http://urbanairship.com/), but could be used as a starting point for your titles within another organization. The actual titles are in [the doc called 'eng'](eng.md), [the doc called 'ops'](ops.md), and [the doc called 'ce'](ce.md).
+This project describes a set of job titles for software engineers, infrastructure admins and engineers, and their managers. It was developed by members of the technology team at [Airship](https://www.airship.com/), but could be used as a starting point for your titles within another organization. The actual titles are in [the doc called 'eng'](eng.md).
```

Comment:
is it me or is the word 'admin' here weirdly trivializing?

Reply:
hmmm yeah let's see if I can clean that up